extension/src/pipeline/util.ts

```diff
@@ -0,0 +1,103 @@
+import {
```

Comment:
[F] For other parts of the codebase I've grouped `quickPick` based functions into their own file (avoiding generic `util` file where possible). E.g. 

- `extension/src/experiments/model/quickPick.ts`
- `extension/src/path/selection/quickPick.ts`
- `extension/src/setup/quickPick.ts`

IMO `parseDataFile` is more of a generic function that could be used elsewhere. I'd probably move that to `extension/src/fileSystem/index.ts` so that other people can find it.

Reply:
I'd always recommend moving functions into a standardized file structure across the project because then we should have a common understanding of where to find things. Not only when we are debugging how the code works but also when we are looking to reuse functions/old pieces of functionality.

Reply:
Makes sense, thanks! Created a `quickPick.ts` instead of `util.ts` and moved the `parseDataFile` to `fileSystem`!